tests/h/services/annotation_metadata_test.py

```diff
@@ -0,0 +1,45 @@
+from unittest.mock import sentinel
+
+import pytest
+
+from h.models import AnnotationMetadata
+from h.services.annotation_metadata import AnnotationMetadataService, factory
+
+
+class TestAnnotationMetadataService:
+    def test_it(self, svc, factories, decrypt_jwe_dict, db_session):
+        decrypt_jwe_dict.side_effect = [{"some": "data"}, {"new": "data"}]
+        anno = factories.Annotation()
+        anno_slim = factories.AnnotationSlim(annotation=anno)
+        db_session.flush()
+
+        svc.set_annotation_metadata_from_jwe(anno, sentinel.jwe)
+
+        decrypt_jwe_dict.assert_called_once_with("TEST_JWE_SECRET_LMS", sentinel.jwe)
+        anno_metadata = (
+            db_session.query(AnnotationMetadata)
+            .filter_by(annotation_id=anno_slim.id)
+            .one()
+        )
+        assert anno_metadata.data == {"some": "data"}
+
+        # Second call updates the existing record
+        svc.set_annotation_metadata_from_jwe(anno, sentinel.jwe)
+        db_session.refresh(anno_metadata)
+        assert anno_metadata.data == {"new": "data"}
+
+    def test_factory(self, pyramid_request):
+        pyramid_request.registry.settings["jwe_secret_lms"] = "TEST_JWE_SECRET_LMS"
+
+        svc = factory(sentinel.context, pyramid_request)
+
+        assert isinstance(svc, AnnotationMetadataService)
+
+    @pytest.fixture
```

Comment:
```suggestion
    @pytest.fixture(autouse=True)
```

... just to make sure no tests in this module accidentally call the real `decrypt_jwe_dict()`, as that can lead to tests passing silently when they should fail